components/dashboard/src/repositories/detail/prebuilds/PrebuildDetailPage.tsx

```diff
@@ -0,0 +1,148 @@
+import { FC, Suspense, useEffect, useMemo } from "react";
+import { useParams } from "react-router";
+import { useConfiguration } from "../../../data/configurations/configuration-queries";
+import { AlertTriangleIcon, CheckCircle2Icon, CircleSlash2Icon, Loader2Icon, Loader2 } from "lucide-react";
+import dayjs from "dayjs";
+import { usePrebuildLogsEmitter } from "../../../data/prebuilds/prebuild-logs-emitter";
+import React from "react";
+import { useToast } from "../../../components/toasts/Toasts";
+import { usePrebuildQuery } from "../../../data/prebuilds/prebuild-query";
+import { LinkButton } from "@podkit/buttons/LinkButton";
+import { repositoriesRoutes } from "../../repositories.routes";
+
+const WorkspaceLogs = React.lazy(() => import("../../../components/WorkspaceLogs"));
+
+interface PageRouteParams {
+    repositoryId: string;
+    prebuildId: string;
+}
+
+export const PrebuildDetailPage: FC = () => {
+    const { repositoryId, prebuildId } = useParams<PageRouteParams>();
+
+    const { data: repository, isLoading: repositoryIsLoading } = useConfiguration(repositoryId);
+    const { toast } = useToast();
+
+    // TODO: watch
+    const { data: prebuild, isLoading: prebuildIsLoading } = usePrebuildQuery(prebuildId);
+
+    const { emitter: logEmitter } = usePrebuildLogsEmitter(prebuildId);
+
+    useEffect(() => {
+        logEmitter.on("error", (err: Error) => {
+            toast("Failed to fetch logs: " + err.message);
+        });
+    }, [logEmitter, toast]);
+
+    const prebuildPhase = useMemo(() => {
+        switch (prebuild?.status?.phase?.name) {
+            case PrebuildPhase_Phase.QUEUED:
+                return {
+                    icon: <Loader2Icon size={20} className="text-gray-500 animate-spin" />,
+                    description: "Prebuild queue",
+                };
+            case PrebuildPhase_Phase.BUILDING:
+                return {
+                    icon: <Loader2Icon size={20} className="text-gray-500 animate-spin" />,
+                    description: "Prebuild building",
+                };
+            case PrebuildPhase_Phase.ABORTED:
+                return {
+                    icon: <CircleSlash2Icon size={20} className="text-gray-500" />,
+                    description: "Prebuild aborted",
+                };
+            case PrebuildPhase_Phase.TIMEOUT:
+                return {
+                    icon: <CircleSlash2Icon size={20} className="text-gray-500" />,
+                    description: "Prebuild timeout",
+                };
+            case PrebuildPhase_Phase.AVAILABLE:
+                return {
+                    icon: <CheckCircle2Icon size={20} className="text-green-500" />,
+                    description: "Prebuild available",
+                };
+            case PrebuildPhase_Phase.FAILED:
+                return {
+                    icon: <AlertTriangleIcon size={20} className="text-kumquat-base" />,
+                    description: "Prebuild available",
```

Comment:
```suggestion
                    description: "Prebuild failed",
```

Reply:
I think after merging my List PR we can unify with https://github.com/gitpod-io/gitpod/pull/19354/files#diff-5374e0e0ce47926ea366af7e62b2a2b1a9c598f1a02c0128631a2fe52fa77251R288